docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
 You can choose which tasks you want to see:
 
 - All open
-- Claimed by me
-- Unclaimed
+- Assigned to me
+- Unassigned
 - Completed
 
-Initially, we have no **Claimed by me** tasks.
+You can also order the tasks on the left menu, clicking on the icon ![order-icon](img/order-icon.png):
 
-![tasklist-claimed-by-me-empty](img/tasklist-claimed-by-me-empty_light.png)
+![tasklist-task-ordering](img/tasklist-task-ordering.png)
+
+### Assign tasks
 
-### Claimed by me tasks
+Initially, we have no **Assigned to me** tasks.
 
-Select the **Unclaimed** list and claim a task using the **Claim** button on the details panel:
+![tasklist-claimed-by-me-empty](img/tasklist-claimed-by-me-empty_light.png)
+
+Select the **Unassigned** list and assign a task using the **Assign to me** button on the top panel:
 
 ![tasklist-claim](img/tasklist-claim_light.png)
 
-### Claim a task
+### Assigned to me tasks
 
-Select the **Claimed by me** list to see if you claimed the task:
+Select the **Assigned to me** list to see if you assigned the task:
```

Comment:
Rephrase to "Select the **Assigned to me** list to see the tasks that are assigned to you. Select a task to work on it."